src/components/DpIcon/DpIcon.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpIcon from './DpIcon.vue'
+
+const meta: Meta<typeof DpIcon> = {
+    component: DpIcon,
+    title: "Components/Icon",
+    argTypes: {}
+}
+
+export default meta
+type Story = StoryObj<typeof DpIcon>
+
+export const Default: Story = {
+    args: {
+        icon: ''
```

Comment:
Shouldn't we pass an icon string here?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/401/commits/265ebb06c9787f16153c8c68b732293f0e152384